components/script/layout_interface.rs

```diff
@@ -95,6 +95,8 @@ pub trait LayoutRPC {
     fn content_box(&self) -> ContentBoxResponse;
     /// Requests the dimensions of all the content boxes, as in the `getClientRects()` call.
     fn content_boxes(&self) -> ContentBoxesResponse;
+    /// Requests the clientTop.
```

Comment:
Not just `clientTop`. I'd say something like "Requests the geometry of this node. This is used by APIs such as `clientTop`."